components/app-gateway/00-init.tf

```diff
@@ -8,9 +8,15 @@ terraform {
     }
   }
 
-  backend "azurerm" {}
+  # backend "azurerm" {}
```

Comment:
```suggestion
  backend "azurerm" {}
```

Reply:
oh was testing it forgot to uncomment that